common/functions/src/aggregates/aggregate_function_factory.rs

```diff
@@ -38,14 +38,12 @@ pub type AggregateFunctionCombinatorCreator = Box<
         + Send,
 >;
 
-lazy_static! {
-    static ref FACTORY: Arc<AggregateFunctionFactory> = {
-        let mut factory = AggregateFunctionFactory::create();
-        Aggregators::register(&mut factory);
-        Aggregators::register_combinator(&mut factory);
-        Arc::new(factory)
-    };
-}
+static FACTORY: Lazy<Arc<AggregateFunctionFactory>> = Lazy::new(|| {
+    let mut factory = AggregateFunctionFactory::create();
+    Aggregators::register(&mut factory);
```

Comment:
Move this into `AggregateFunctionFactory::create()` is better?

Reply:
I think a bit hard:
Lazy<Arc<AggregateFunctionFactory>> is initialed global.

If we move it to `AggregateFunctionFactory::create()`, maybe we should call it in the first entry point.

Reply:
Maybe `once` privimite can be used here